src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.module.css

```diff
@@ -0,0 +1,63 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 30px 1fr 30px;
+    grid-template-rows: auto;
+  }
+}
+
+.carouselWrapper {
+  grid-column: 2;
+}
+
+@media (--tablet-and-down) {
+  .carouselWrapper {
+    column-gap: 10px;
+    grid-column: 1/ span 13;
+    grid-column: 1 / span 13;
+    grid-template-columns: repeat(13, minmax(0, 1fr));
```

Comment:
A few things here:

1. `grid-template-columns` and `column-gap` do nothing, since this is not a grid parent.
2. `grid-column` is being set twice.
3. `grid-column` is being set to span 13 columns, even though there are only three. Keep in mind that `grid-column` is set on the grid child, so the column spec refers to the parent element's columns, which in this case is the `.bleedWrapper`, which only has three columns.
4. If you go with my advice above of changing the `.bleedWrapper` to a `display: block` element in the mobile view, then you don't need to set any of this.

Reply:
👍 Ah, the display block makes it much easier. Thought the mobile view needs to be in a grid.